README.md

```diff
@@ -116,7 +116,7 @@ To download and extract [a multilingual model](https://github.com/huji-nlp/tupa/
 
 Run the parser using the model:
 
-    python -m tupa example.txt --lang [example lang] -m models/bert_multilingual_layers_4_layers_pooling_weighed_align_sum --use-bert --bert-model bert-base-multilingual-cased --bert-layers -1 -2 -3 -4 --bert-layers-pooling weighed --bert-token-align-by sum --bert-multilingual 0
+    python -m tupa example.txt --lang [example lang] -m bert_multilingual_layers_4_layers_pooling_weighed_align_sum --use-bert --bert-model bert-base-multilingual-cased --bert-layers -1 -2 -3 -4 --bert-layers-pooling weighed --bert-token-align-by sum --bert-multilingual 0
```

Comment:
Actually, shouldn't the bert-specific paramters be saved to the model .json and loaded in model.py load()?
Also, `[example lang]` is not a language; should be explained.

Reply:
I was planning on doing this in the "final version" (with xlm, leno, etc.), but it's probably better to so now, as people who are not familiar with the code might use it. I'll work on it.

Reply:
How will it work if the params will be loaded from model.json but the user also pass the argument in the command?

Reply:
The JSON will have to override the user since we can't distinguish whether Argparser's attributes are from the command line or from the default values.

Reply:
Adding new commit.
Tested on:
1) Creating a new mode without bert and reloading it.
2) Creating a bert model and reloading it
3) reloading a model created from the vanilla tupa code